test/parallel/test-http-server-write-end-after-end.js

```diff
@@ -6,20 +6,23 @@ const http = require('http');
 const server = http.createServer(handle);
 
 function handle(req, res) {
-  res.on('error', common.mustCall((err) => {
-    common.expectsError({
-      code: 'ERR_STREAM_WRITE_AFTER_END',
-      name: 'Error'
-    })(err);
-    server.close();
-  }));
+  res.on('error', common.mustNotCall());
 
   res.write('hello');
   res.end();
 
-  setImmediate(common.mustCall(() => {
+  setImmediate(() => {
+    // TODO: Known issues. OutgoingMessage does not always invoke callback
+    // in end. Also streams don't propagate write after end to callback.
```

Comment:
https://github.com/nodejs/node/issues/33684

Reply:
I would say that should be fixed first.

Reply:
I still think https://github.com/nodejs/node/issues/33684 should be fixed before landing this.

Reply:
@mcollina: That is an existing issue that's been there for a long time (i.e. not a regression). Why do you feel that should block this?